src/main/java/org/commonjava/indy/service/archive/services/LogService.java

```diff
@@ -0,0 +1,276 @@
+            reader.beginObject();
+            while (reader.hasNext()) {
+                items.add(readTraceLogItems(reader));
+            }
+            reader.endObject();
+        }
+        reader.endArray();
+
+        return items;
+    }
+
+    private TraceLogItem readTraceLogItems(JsonReader reader) throws IOException {
+        TraceLogItem item = new TraceLogItem();
+
+        while (reader.hasNext()){
+            String jsonKey = reader.nextName();
+            switch (jsonKey) {
+                case "storeKey":
+                    item.setStoreKey(reader.nextString());
+                    break;
+                case "accessChannel":
+                    item.setAccessChannel(reader.nextString());
+                    break;
+                case "path":
+                    item.setPath(reader.nextString());
+                    break;
+                case "localUrl":
+                    item.setLocalUrl(reader.nextString());
+                    break;
+                case "md5":
+                    item.setMd5(reader.nextString());
+                    break;
+                case "sha256":
+                    item.setSha256(reader.nextString());
+                    break;
+                case "sha1":
+                    item.setSha1(reader.nextString());
+                    break;
+                case "size":
+                    item.setSize(reader.nextLong());
+                    break;
+                default:
+                    reader.skipValue();
+                    break;
+            }
+        }
+
+        return item;
+    }
+
+
+
+    public static class TraceLog{
+        @JsonProperty("key")
+        private Map<String, String> keys;
+        @JsonProperty("uploads")
+        private List<TraceLogItem> uploads;
+        @JsonProperty("Downloads")
+        private List<TraceLogItem> downloads;
+
+        public TraceLog() {
+            this.keys = new HashMap<>();
+            this.uploads = new ArrayList<TraceLogItem>();
+            this.downloads = new ArrayList<TraceLogItem>();
+        }
+
+        @Override
+        public String toString() {
+            return "TraceLog{" +
+                    "keys=" + keys +
+                    ", uploads=" + uploads +
+                    ", downloads=" + downloads +
+                    '}';
+        }
+
+        public Map<String, String> getKeys() {
+            return keys;
+        }
+
+        public void setKeys(Map<String, String> keys) {
+            this.keys = keys;
+        }
+
+        public List<TraceLogItem> getUploads() {
+            return uploads;
+        }
+
+        public void setUploads(List<TraceLogItem> uploads) {
+            this.uploads = uploads;
+        }
+
+        public List<TraceLogItem> getDownloads() {
+            return downloads;
+        }
+
+        public void setDownloads(List<TraceLogItem> downloads) {
+            this.downloads = downloads;
+        }
+    }
+
+
+    public static class TraceLogItem {
```

Comment:
Recommend to extract this as a separate class but not embedded in other class. And seems this is indy's folo tracking record, so recommend to use "TrackedContentEntry" as class name to apply same with indy. See: https://github.com/Commonjava/indy/blob/master/addons/folo/model-java/src/main/java/org/commonjava/indy/folo/model/TrackedContentEntry.java